apps/engine/test/engine/db/block_test.exs

```diff
@@ -698,4 +698,34 @@ defmodule Engine.DB.BlockTest do
         :no_gas_reference
     end
   end
+
+  defp vault_response_mock() do
```

Comment:
exvcr too heavy?

Reply:
or you can use FakeServer. It's already added to the childchain

Reply:
I don't see a particular reason why I would add another layer of mocking (exvcr or FakeServer).